pages/common/prosodyctl.md

```diff
@@ -0,0 +1,23 @@
+# prosodyctl
+
+> The control tool for the prosody XMPP server.
+
+- Show the status of the prosody server:
+
+`sudo prosodyctl status`
```

Comment:
Good question! I'm not sure either, but it appears to be because it needs access to it's configuration file - which is by default stored in a directory that's got the permissions 0750, and is owned by `prosody:prosody`.